src/deepsparse/loggers/build_logger.py

```diff
@@ -39,6 +40,7 @@
     SystemLoggingGroup,
 )
 from deepsparse.loggers.helpers import get_function_and_function_name
+from deepsparse.loggers.metric_functions.registry import DATA_LOGGING_REGISTRY
```

Comment:
will the registry be populated here if importing directly from ` deepsparse.loggers.metric_functions.registry` want to make sure it is by the time it is called (rather than depending on going through `get_function_and_function_name`

Reply:
yes, it will. on the import, the `DATA_LOGGING_REGISTRY` "goes through" all its decorators and registers all the appropriate functions